pkg/resources/pullSecret.go

```diff
@@ -16,40 +18,38 @@ const (
 	DefaultOriginPullSecretNamespace = "openshift"
 )
 
-// Gets the default pull secret for pulling container images from registry
-func GetDefaultPullSecret(context context.Context, client k8sclient.Client, inst *integreatlyv1alpha1.Installation) (corev1.Secret, error) {
+// Copys the default pull secret to a target namespace
```

Comment:
For exported functions & variables, the Go convention is to begin the comment with the exported name, e.g.:

```go
// CopyDefaultPullSecretToNameSpace copies the default pull secret to a target namespace
```

Reply:
Thanks, fixed this now.